src/material.h

```diff
@@ -75,4 +77,13 @@ class material_type
         const mat_burn_data &burn_data( size_t intensity ) const;
 };
 
+namespace materials
```

Comment:
> This variant exists in the code, and it's IMO better than the rest.

If it does how common actually is it? The former @BevapDin mentioned is abundant.

> It's about clarity and predictability

It's not predictable if it's different to most of the other code. I'd agree with the below that its less clear.

> This seems to be better as it avoids some potential confusion about what scopes (the namespace, the class) contains what

> No, whats the need?

Could your PR be merged without that code. We don't currently have any need for separating the functionality so unless you can prove otherwise there is no need. This is the only objective test that can be made.